modules/kinesis/src/it/scala/com/snowplowanalytics/snowplow/enrich/kinesis/Containers.scala

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2022-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.kinesis
+
+import org.slf4j.LoggerFactory
+
+import org.testcontainers.containers.{BindMode, GenericContainer => JGenericContainer, Network}
+import org.testcontainers.containers.wait.strategy.Wait
+import org.testcontainers.containers.output.Slf4jLogConsumer
+
+import com.dimafeng.testcontainers.{FixedHostPortGenericContainer, GenericContainer}
+
+object Containers {
+
+  val (localstack, localtackNetwork) = {
+    val container = FixedHostPortGenericContainer(
+      imageName = "localstack/localstack-light:1.0.4",
```

Comment:
[Latest is v1.2](https://github.com/localstack/localstack/releases/tag/v1.2.0). Not sure if we'll get much benefit from bumping but if it doesn't break anything might as well.